app/mailers/comment_mailer.rb

```diff
@@ -8,4 +8,12 @@ def new_comment(user, comment)
       subject: 'New massege under your post'
     )
   end
+
+  def notify_user(user, comment)
```

Comment:
bad naming
why do you create new mailer action if it work the same as previous? maybe it would be better to show all unseen comments in on email?